Readme.md

```diff
@@ -1,18 +1,10 @@
 # VFS for Git
 
-## Windows

```

Comment:
Maybe mention in the commit message that a Mac version used to be planned, right until changes in Catalina made it virtually impossible?